packages/toolkit/src/amazonqFeatureDev/session/session.ts

```diff
@@ -51,7 +51,7 @@ export class Session {
             await this.setupConversation(msg)
             this.preloaderFinished = true
 
-            telemetry.amazonq_startChat.emit({
+            telemetry.amazonq_startConversationInvoke.emit({
```

Comment:
See: https://github.com/aws/aws-toolkit-common/pull/693. Looks like another team removed the `amazonq_startChat` metric in favor of this one, but didn't update it here so I'm doing it to avoid a compilation error. I updated the telemetry package version because I made my own updates to our metrics.

Reply:
blocker.
this metrics already emitted on L3.
You can fully delete this metric here as in this CR: 
https://github.com/aws/aws-toolkit-vscode-staging/pull/1578/files

Reply:
Done